UnityProject/Assets/Mirror/Runtime/NetworkClient.cs

```diff
@@ -988,7 +988,7 @@ internal static void ApplySpawnPayload(NetworkIdentity identity, SpawnMessage me
             ///CUSTOM UNITYSTATION CODE///
             //Mirror specifically says that they dont support NetworkIdentities nested in normal gameobjects.
             //But, we do it, so we have to override this and make sure we read and send world pos
```

Comment:
```suggestion
            //But, we do it, so we have to override this and make sure we read and send local pos
```

Does it need to be that now?